packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -39,56 +49,196 @@ class PoolMaster extends Component {
-      <SelectFiles
-        onChange={file => installSupplementalPackOnAllHosts(pool, file)}
-      />
-    </Upgrade>
-  </div>
-))
+})
+export default class TabAdvanced extends Component {
+  _closeEditRemoteSyslogForm = () => this.setState({ editRemoteSyslog: false })
+  _closeEditRemoteSyslogHostsForm = () =>
+    this.setState({ editRemoteSyslogHosts: false })
+  _openEditRemoteSyslogHostsForm = idHost =>
+    this.setState({ idHost, editRemoteSyslogHosts: true })
+
+  _setNameLabel = (host, nameLabel) => editHost(host, { name_label: nameLabel })
+
+  render () {
+    const { hostsByPool, gpuGroups, pool } = this.props
+    const { editRemoteSyslog, editRemoteSyslogHosts, idHost } = this.state
+    const hostsOfPool = sortBy(hostsByPool[pool.uuid], 'name_label')
+
+    const syslogRemoteHost =
+      uniq(map(hostsOfPool, host => host.logging.syslog_destination)).length ===
+      1
+        ? hostsOfPool[0].logging.syslog_destination
+        : ''
+
+    return (
+      <div>
+        <Container>
+          <Row>
+            <Col>
+              <h3>{_('xenSettingsLabel')}</h3>
+              <table className='table'>
+                <tbody>
+                  <tr>
+                    <th>{_('uuid')}</th>
+                    <Copiable tagName='td'>{pool.uuid}</Copiable>
+                  </tr>
+                  <tr>
+                    <th>{_('poolHaStatus')}</th>
+                    <td>
+                      {pool.HA_enabled
+                        ? _('poolHaEnabled')
+                        : _('poolHaDisabled')}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('setpoolMaster')}</th>
+                    <td>
+                      <PoolMaster pool={pool} />
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('poolRemoteSyslog')}</th>
+                    <td>
+                      <SingleLineRow>
+                        <Col>
+                          <span>{syslogRemoteHost} </span>
+                          <ActionRowButton
+                            btnStyle={editRemoteSyslog ? 'info' : 'primary'}
+                            handler={this.toggleState('editRemoteSyslog')}
+                            icon='edit'
+                            tooltip={_('poolEditRemoteSyslog')}
+                          />
+                        </Col>
+                      </SingleLineRow>
+                      {editRemoteSyslog && (
+                        <Row>
+                          <Col>
+                            <EditPoolRemoteSyslog
+                              hosts={hostsOfPool}
+                              onClose={this._closeEditRemoteSyslogForm}
+                            />
+                          </Col>
+                        </Row>
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('syslogRemoteHost')}</th>
```

Comment:
Why 2 `tr`s for (almost) the same thing?
Either handle the case when all the logging hosts are the same:
  - List of editable if they're different
  - "Syslog host: <myHost>" if they're the same

or simply list them all everytime if you like it better.